rook/migration/v1.12/README.md

```diff
@@ -0,0 +1,61 @@
+# Upgrade rook-ceph to v1.12.x
+
+## Prerequisites
+
+- [ ] Check the state of rook-ceph so it is healthy before you begin:
+
+  ```bash
+  helm -n rook-ceph list --all
+  # should show all deployed
+  kubectl -n rook-ceph get jobs
+  # should show all completed
+  kubectl -n rook-ceph get pods
+  # should show all running or completed
+  kubectl -n rook-ceph get cephclusters
+  # should show PHASE = Ready and HEALTH = HEALTH_OK
+  ```
+
+## Apply upgrade - _disruptive_
+
+### **1. Pull chart version**
+
+```console
+git clone --single-branch --branch v1.12.11 https://github.com/rook/rook.git
+sudo cp -rf rook/deploy/charts/* helmfile.d/upstream/
+rm -rf rook
```

Comment:
1. Could we rework this to fetch it from their helm repository instead?
2. We should _absolutely_ not need `sudo` for this.

Reply:
Example using helm to fetch and untar a helm chart
```bash
helm fetch rook-release/rook-ceph-cluster --version v1.12.11 --untar
```